examples/frontend/containers/Layout.tsx

```diff
@@ -41,8 +43,12 @@ export default function Layout({ children, project = 'share' }) {
         }
     }
 
-    const preSignIn = () => {
+    const preSignIn = async () => {
         const project = mapPathToProject(router.asPath)
+        if(localStorage.getItem('contractAddress')){
+            localStorage.removeItem('contractAddress')
+            await signOut();
+        }
         localStorage.setItem('contractAddress', project)
```

Comment:
Is the the if statement and `localStorage.removeItem` necessary here? Because I think `localStorage.setItem` will overwrite the previous value anyway

Reply:
Yeah. we can remove that.